composer.json

```diff
@@ -4,10 +4,10 @@
     "description": "Site package for the Neos Demo Site",
     "license": "GPL-3.0+",
     "require": {
-        "neos/neos": "*",
-        "neos/nodetypes": "*",
-        "typo3/neos-seo": "dev-master",
-        "flowpack/neos-frontendlogin": "dev-master"
+        "neos/neos": "~3.1",
```

Comment:
In 3.0 we also use that. And anything > 3.1 should be compatible by semver definition…